custom/icds_reports/reports/governance_apis.py

```diff
@@ -128,7 +128,18 @@ def get_cbe_data(length, year, month, order, query_filters):
     data = AggGovernanceDashboard.objects.filter(
         month=date(year, month, 1),
         **query_filters
-    ).order_by(*order).values(
+    ).order_by(*order).annotate(
```

Comment:
These columns are necessary for the API? as said before they don't actually provide any information to the recipient. This is a nifty trick though which I had never seen before

Reply:
Yes, This field is needed in the APi.